src/server/scripts/Northrend/Ulduar/Ulduar/boss_flame_leviathan.cpp

```diff
@@ -368,11 +367,12 @@ class boss_flame_leviathan : public CreatureScript
                 else if (_speakTimer > 41000 && _speakTimer < 60000)
                 {
                     me->RemoveFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NON_ATTACKABLE);
-                    me->SendMonsterMove(380.4f, -14.3f, 409.8f, 2000);
-                    me->UpdatePosition(380.4f, -14.3f, 409.8f, me->GetOrientation());
+                    TurnGates(true, false);
+                    me->MonsterMoveWithSpeed(322.39f, -14.5f, 409.8f, 100.0f);
+                    me->UpdatePosition(322.39f, -14.5f, 409.8f, me->GetOrientation());
```

Comment:
Move the magic numbers to a proper position variable, please.

Reply:
done.